notebooks/Emukit-task-spread-of-disease-with-seir-model.ipynb

```diff
@@ -0,0 +1,908 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Spread of an infectious disease: SEIR-model with Emukit\n",
+    "\n",
+    "This notebook uses the SEIR-model and Emukit to answer several application driven research question."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# General imports\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import colors as mcolors\n",
+    "\n",
+    "# Figure config\n",
+    "colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)\n",
+    "LEGEND_SIZE = 15\n",
+    "TITLE_SIZE = 25\n",
+    "AXIS_SIZE = 15\n",
+    "FIGURE_SIZE = (12, 8)\n",
+    "\n",
+    "# for reproducability\n",
+    "np.random.seed(42)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The SEIR epidemic model\n",
+    "\n",
+    "The [SEIR model](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology#The_SEIR_model) is a compartmental model of biology that simulates the spread of an infectious disease. \n",
+    "A population of $N$ individuals is divided into the compartments: **S**: susceptible, **E**: exposed, **I**: infected, **R**: recovered with $N = N_S + N_E + N_I + N_R$.\n",
+    "The compartment 'exposed' is introduced to model incubation times; exposed objects are already infected, but are not infectious themselves yet.\n",
+    "\n",
+    "#### High-fidelity model \n",
```

Comment:
Before jumping into explanations of high and low fidelity models, perhaps we could talk a bit about what will happen in the notebook? As in, what are we interested in, which questions we ask, how are we going to approach answering them?

You actually have that later in the notebook, at least the questions. I would consider changing order of paragraphs here a bit

Reply:
I can put the high-fidelity- Low fidelity distinction and '4. Other' task after the model introduction and the first 3 tasks. I kind of need the notation from the model to describe the tasks 1-3.